src/user/dashboard/news-feed/popups/SharePost.js

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect } from "react";
+import { Button, Modal } from "react-bootstrap";
+import PropTypes from "prop-types";
+import {
+  FacebookShareButton,
+  FacebookIcon,
+  LinkedinShareButton,
+  LinkedinIcon,
+  TwitterShareButton,
+  TwitterIcon,
+  WhatsappShareButton,
+  WhatsappIcon,
+  TelegramShareButton,
+  TelegramIcon,
+  RedditShareButton,
+  RedditIcon,
+} from "react-share";
+
+const SharePostModal = (props) => {
+  useEffect(() => {
+    console.log("useEffect from share-post ", props);
+  }, [props]);
+
+  const url = "https://donut.codeuino.org/";
```

Comment:
@AuraOfDivinity The url to share should be the post's url that the user will be sharing